Assignment #7, Issue #83 | added Michigan directory | added bliss-mi and waukesha -wi .ipynb and .csv files

lrwives opened the pull request:
@nkorinek, @lisestdenis , and @lwasser, 

In this pr I have:

- created the directory united-state-of-america/michigan

- added the notebook titled bliss-mi.ipynb to the united-state-of-america/michigan directory

- added the notebook titled waukesh-wi.ipynb to the united-state-of-america/wisconsin directory

- added the files bliss-mi.csv and waukesha-wi.csv to the 0-csv-files directory

This pr addresses issue #83 .

lrwives committed:
add 0-csv-files/bliss-mi.csv, 0-csv-files/waukesha-wi.csv, united-state-of-america/wisconsin/waukesha-wi.ipynb

lrwives committed:
Create michigan

add Michigan folder

lrwives committed:
Delete michigan

lrwives committed:
Create bliss-mi.ipynb

lrwives committed:
add blii-mi.ipynb

lrwives commented:
Apologies for the awkward/misspelled push messages from git bash. That's something I will work on doing better in future assignments.

lisestdenis reviewed (changes requested):
Hi @lrwives 

Can you make the following changes:
update your type options to match one of the available options (e.g. place I've lived)
create city directory for your notebook files and re-submit.

@lisestdenis

lrwives committed:
requested edits made to bliss and waukesah *.csv files

lrwives commented:
Hi @lisestdenis 

 I updated the csv files so that the "type" category matches the available options. I also changed the names of the csv files to inclued "united_states" at the beginning.

Can you clarify what you mean by "creating city directory for the notebook files"? I added my jupyter notebook files to the wisconsin and michigan directories, respectively. 

Thanks,
@lrwives

lisestdenis committed:
Delete bliss-mi.csv

lisestdenis committed:
Delete waukesha-wi.csv
